styles/variables-silicon.less

```diff
@@ -36,8 +36,12 @@
 
 // CheckboxItem
 // ---------------------------------------
+@agate-checkboxitem-border-width: 1px;
 @agate-checkboxitem-icon-margin: 9px;
-@agate-checkboxitem-font-size: 30px;
+@agate-checkboxitem-font-size: 28px;
+@agate-checkboxitem-selected-before-content: "•";
```

Comment:
Interesting!
It might be same GUI in ToggleButton, right? But the approach to implement it is different. Please check the comment - https://github.com/enactjs/agate/pull/262/files#diff-f78bc3f30e4ce632324db4a14eeb3208R172. I think that it would be better to have the same approach. Please let me know your opinion.

Reply:
CC: @daniel-stoian-lgp

Reply:
Yes, I agree. I created a general Indicator styling for that bullet, as it is used with each type of list item.